src/main/webapp/script.js

```diff
@@ -12,6 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let map;
+/* Editable marker that displays when a user clicks in the map. */
+let editMarker;
+
```

Comment:
usually with js there's a problem with ```let``` so it'd be better to use ```var```

Reply:
This is what they had in their example but I might change it

Reply:
i'm not 100% sure with let vs var, but if the example has let and it works, then carry on